sources/newRif/post.py

```diff
@@ -1,40 +1,87 @@
+import requests
 import django
-django.setup()
 import json
+import csv
+import re
+django.setup()
 from sefaria.model import *
 from sources import functions
+from sefaria.utils.talmud import daf_to_section, section_to_daf
 from rif_utils import tags_map, path, get_hebrew_masechet
 
-server = 'http://localhost:8000'
-#server = 'https://glazner.cauldron.sefaria.org'
+#server = 'http://localhost:8000'
+server = 'https://glazner.cauldron.sefaria.org'
 
+def housekeep(string):
+    string = re.sub(r'\s', ' ', string)
+    string = re.sub(' +', ' ', string)
+    string = re.sub(r' ([\)\]:\.])', r'\1 ', string)
+    string = re.sub(r'([\(\[]) ', r' \1', string)
+    string = re.sub('(<sup.*?/i>) ', r' \1', string)
+    string = re.sub(' +', ' ', string)
+    return string
+
+def post_rif(masechtot=list(tags_map), index=True, text=True, links=True, delete=True, server = 'http://localhost:8000'):
+    for masechet in masechtot:
+        title = 'Rif ' + masechet
+        if delete:
+            try:
+                library.get_index(title).delete()

```

Comment:
This line is a bit deceptive. The method is designed to post to any server, but deleting will only delete the local version. It's important to remember that this method will have drastically different effects when posting locally and posting to a remote server. I'd recommend moving the delete logic outside of the posting logic.